prime-router/src/main/kotlin/azure/SendFunction.kt

```diff
@@ -22,11 +22,11 @@ import java.util.logging.Level
  */
 const val dataRetentionDays = 7L
 const val send = "send"
-const val maxRetryCount = 10
+const val maxRetryCount = 4
```

Comment:
Changed to 4 retries instead of 10.  Occasionally we need one retry in redox.  Retries in sftp have never worked.